zstacklib/ansible/zstacklib.py

```
@@ -2576,6 +2578,19 @@ def generate_163_yum_repo(self):
     def generate_aliyun_yum_repo(self):
         self.generate_yum_repo_config_from_zstack_lib("zstack-aliyun-yum.repo")
 
+    def generate_zstack_local_yum_repo(self):
+        generate_zstack_local_repo_command = """
+cat > /etc/yum.repos.d/zstack-local.repo << EOF
+[zstack-local]
+name=zstack-local
+baseurl=file:///opt/zstack-dvd/\$basearch/\$YUM0
```

Ruff (0.15.12)

[warning] Invalid escape sequence: `\$`

Use a raw string literal

(W605)